operator/controllers/keptnapp/controller.go

```diff
@@ -191,7 +191,7 @@ func (r *KeptnAppReconciler) deprecateAppVersions(ctx context.Context, app *klcv
 		}
 
 		if err != nil {
-			r.Log.Error(err, "AppVersion not found")
+			r.Log.Error(err, "Could not get KeptnAppVersion")
```

Comment:
I would rephrase it as 

if err!=nil {
    if ! errors.IsNotFound {
        // code
    }
}

also you could consider to wrap everything into a helper function instead of having resultErr and continue

Reply:
I don't get it, I want to continue also when error occures